packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/proxies/dataservice/ArtifactProxy.java

```diff
@@ -19,30 +20,36 @@
 public interface ArtifactProxy {
 
 	@GET
+	@LogRestClientTime
 	List<Artifact> getArtifacts( @DefaultValue("100") @QueryParam("page_size") Integer pageSize,
 															 @DefaultValue("0") @QueryParam("page") Integer page);
 
 	@POST
+	@LogRestClientTime
```

Comment:
All of these annotations are using the default grouping on the log i think - is that intentional? Should we define different groups?

Reply:
I think those could be define in another PR after the hackathon or if MITRE requires some specifics grouping.

Reply:
Agreed.  Someone that knows the systems better than I do should go in and label them

Reply:
Sounds good!